From 07ce9278fbfc06ce14cd16d463788c55442ee163 Mon Sep 17 00:00:00 2001
From: Michal Sekletar <msekleta@redhat.com>
Date: Wed, 1 Apr 2015 13:08:25 +0200
Subject: [PATCH] mount: fix up wording in the comment

(cherry picked from commit 340a1d2330ddc1dd18ad75bcdddf32f63c84b4a1)

Cherry-picked from: 375af09
Resolves: #1222517
---
 src/core/mount.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/core/mount.c b/src/core/mount.c
index 3359220..fd4fb6f 100644
--- a/src/core/mount.c
+++ b/src/core/mount.c
@@ -108,7 +108,7 @@ static bool needs_quota(const MountParameters *p) {
         assert(p);
 
         /* Quotas are not enabled on network filesystems,
-         * but we them, for example, on storages connected via iscsi */
+         * but we want them, for example, on storage connected via iscsi */
         if (p->fstype && fstype_is_network(p->fstype))
                 return false;
 
